src/main/java/de/muenchen/isi/domain/service/AbfrageService.java

```diff
@@ -237,6 +237,18 @@ public void deleteInfrasturkturabfrageById(final UUID id)
         this.infrastrukturabfrageRepository.deleteById(id);
     }
 
+    /**
+     * Die Methode gibt alle {@link InfrastrukturabfrageModel} als Liste zurück welche einem Bauvorhaben zugeordnet sind.
+     *
+     * @param bauvorhabenId zum Identifizieren des {@link BauvorhabenModel}
+     * @return Liste von {@link InfrastrukturabfrageModel} welche einem Bauvorhaben zugeordent sind
+     */
+    public List<InfrastrukturabfrageModel> getAllReferencedAbfragenForBauvorhaben(final UUID bauvorhabenId) {
```

Comment:
Kann dieser Code gleich in die Methode `getAbfrageListElementsThatReferenceBauvorhaben` oder wird die Extraktion wiederverwendet?